en/docs/administer/managing-users-and-roles/managing-user-roles.md

```diff
@@ -149,55 +149,61 @@ This will update all scope mappings in tenant-conf.json with Internal/creator as
-    | apim:mediation_policy_manage           | &check; |                      | &check;            |                       |                      |                     |
-    | apim:threat_protection_policy_create   | &check; |                      | &check;            |                       |                      |                     |
-    | apim:ep_certificates_add               | &check; |                      | &check;            |                       |                      |                     |
-    | apim:ep_certificates_view              | &check; |                      | &check;            |                       |                      |                     |
-    | apim:api_key                           | &check; |                      | &check;            |                       |                      |                     |
-    | apim_analytics:admin                   | &check; |                      |                    |                       |                      |                     |
-    | apim_analytics:product_manager         | &check; | &check;              | &check;            | &check;               |                      |                     |
-    | apim_analytics:api_developer           | &check; | &check;              | &check;            |                       |                      |                     |
-    | apim_analytics:app_developer           | &check; |                      |                    | &check;               |                      |                     |
-    | apim_analytics:devops_engineer         | &check; | &check;              | &check;            | &check;               |                      |                     |
-    | apim_analytics:analytics_viewer        | &check; |                      |                    |                       | &check;              |                     |
-    | apim_analytics:everyone                |         |                      |                    |                       |                      | &check;             |
-    | apim:pub_alert_manage                  | &check; |                      | &check;            |                       |                      |                     |
-    | apim:sub_alert_manage                  | &check; |                      |                    | &check;               |                      |                     |
-    | apim:tenantInfo                        | &check; |                      |                    |                       |                      |                     |
-    | apim:admin_operations                  | &check; |                      |                    |                       |                      |                     |
+    | Role                                          | admin   | Internal/publisher   | Internal/creator   | Internal/subscriber   | Internal/analytics   | Internal/everyone   |
```

Comment:
Shouldn't the title here be "scope" instead of "role" ?

Reply:
Can we categorize the follows as publisher/ store/admin and analytics related scopes

Reply:
> Shouldn't the title here be "scope" instead of "role" ?

Yes, the column contains scopes that should be refactored.
Fixed with https://github.com/wso2/docs-apim/pull/1690/commits/c1ad5f61a5bb3612d1a91560ab6823ddb729bcc8